### PR TITLE
Validate master/publicMaster args to create-master-certs

### PR DESCRIPTION
**Pull request by liggitt**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1305165

### CODE REVIEW

**Comment by liggitt** [comment]
[test]

File: pkg/cmd/server/admin/create_mastercerts.go [hunk 1]
```diff
@@ -118,13 +119,26 @@ func (o CreateMasterCertsOptions) Validate(args []string) error {
 		return errors.New("at least one hostname must be provided")
 	}
 	if len(o.CertDir) == 0 {
-		return errors.New("cert-dir must be provided")
+		return errors.New("--cert-dir must be provided")
```
Comment (deads2k) [inline pkg/cmd/server/admin/create_mastercerts.go]: Matching flag names is logically a coincidence.  Consider the composition case.

**Comment by deads2k** [comment]
specifying flags in the validation for an options object isn't always right, but I don't feel super strongly.

lgtm otherwise.

**Comment by liggitt** [comment]
[merge]

**Pull request merged**